lib/components/app/print-layout.js

```diff
@@ -82,13 +99,15 @@ class PrintLayout extends Component {
         }
 
         {/* The main itinerary body */}
-        {itinerary
-          ? <PrintableItinerary
-            configCompanies={configCompanies}
-            customIcons={customIcons}
-            itinerary={itinerary}
-            timeFormat={timeFormat} />
-          : null
+        {itinerary &&
+          <div>
```

Comment:
Maybe we should use a [fragment](https://reactjs.org/docs/fragments.html) here instead of a div.
```
<>
           <PrintableItinerary
              config={config}
              itinerary={itinerary}
              LegIcon={this.customLegIcon(customIcons)}
            />
            <TripDetails itinerary={itinerary} />
</>
```

Reply:
Good suggestion.